src/frontend/pages/AllBookings.tsx

```diff
@@ -66,82 +71,23 @@ export const AllBookings: React.FC<RouteComponentProps> = () => {
+      <div className="table-responsive">
+        <BookingList
+          bookings={upcomingBookings}
+          fetchBookings={fetchBookings}
+          setBookings={setBookings}
+          handleDeleteBooking={handleDeleteBooking}
+        />
+      </div>
+      <h2>Past Bookings</h2>
       <div className="table-responsive">
-        <table>
-          <thead>
-            <tr>
-              <th>Type</th>
-              <th>Date</th>
-              <th>Slot time</th>
-              <th>Booking details</th>
-              <th style={{ width: '18rem' }}>Note</th>
-              <th>User email</th>
-              <th></th>
-            </tr>
-          </thead>
-          <tbody>
-            {bookings.map(
-              ({
-                uuid,
-                end,
-                start,
-                fullName,
-                phone,
-                email,
-                bookingType,
-                bookingNote,
-                user,
-                workingRemotely,
-              }) => {
-                return (
-                  <tr key={uuid}>
-                    <td>{bookingType.name}</td>
-                    <td>{moment(start).format('ddd Do MMM YYYY')}</td>
-                    <td>
-                      {moment(start).format('HH:mm')} - {moment(end).format('HH:mm')}
-                    </td>
-                    <td>
-                      {fullName}
-                      <br />
-                      {email}
-                      <br />
-                      {phone}
-                      <br />
-                      Work location: {workingRemotely ? 'Remote' : 'Office'}
-                    </td>
-                    <td>
-                      <UpdateBookingNoteForm
-                        fetchBookings={fetchBookings}
-                        setBookings={setBookings}
-                        booking={{
-                          uuid,
-                          end,
-                          start,
-                          fullName,
-                          phone,
-                          email,
-                          bookingType,
-                          bookingNote,
-                          user,
-                          workingRemotely,
-                        }}
-                      />
-                    </td>
-                    <td>{user.email}</td>
-                    <td>
-                      <button
-                        className="button button-xxs button-border button-error"
-                        onClick={handleDeleteBooking(uuid)}
-                      >
-                        Delete
-                      </button>{' '}
-                    </td>
-                  </tr>
-                );
-              },
-            )}
-          </tbody>
-        </table>
+        <BookingList
+          bookings={pastBookings}
+          fetchBookings={fetchBookings}
+          setBookings={setBookings}
+          handleDeleteBooking={handleDeleteBooking}
+        />
```

Comment:
This past bookings list could be in reverse order so that the latest bookings are listed first, I started to look at whether that is possible to define with react-data-table but ended up implementing it already. Check my commit that I didn't do anything wrong

Reply:
Looks good, and I also update the sorting for the past bookings under _User Profile_ page